webapp/src/modules/vendor/decentraland/nft/api.ts

```diff
@@ -235,6 +236,26 @@ class NFTAPI extends BaseAPI {
 
     return queryParams.toString()
   }
+
+  async getOwners( params: OwnersFilters): Promise<{data: OwnersResponse[], total: number}> {
```

Comment:
```suggestion
  async getOwners( params: OwnersFilters): Promise<{ data: OwnersResponse[], total: number }> {
```
can we run prettier in this file please :) ?